src/main/scala/Driver.scala

```diff
@@ -0,0 +1,73 @@
+import com.databricks.spark.avro._
+import org.apache.spark.SparkConf
+import org.apache.spark.sql.functions._
+import org.apache.spark.sql.{Column, SparkSession}
+
+object Driver {
+  def main(args: Array[String]): Unit = {
+    val conf: SparkConf = new SparkConf().setMaster("local[*]").setAppName("Spark-Assignment")
+    val spark: SparkSession = SparkSession.builder().config(conf).getOrCreate()
+
+    val schemaText = spark.sparkContext.wholeTextFiles("./src/main/resources/outputSchema.avsc").collect()(0)._2
+    val inputPath_4 = "src/main/resources/part-00000-a4794630-9584-4152-b5ed-595fd7608322.avro"
+    val inputPath_212 = "src/main/resources/part-00000-64940e90-dcc3-4d23-9a38-c7e1e1d24820.avro"
+    val inputPath_316 = "src/main/resources/part-00000-a3521293-47b8-4b2c-983a-2b973319fd51.avro"
```

Comment:
Why are you reading only a single file for each DPID? There are multiple files/folders right?

Reply:
Read all remaining files